src/pages/ide/index.md

```diff
@@ -1 +1,5 @@
 # Integrated Development Environments
```

Comment:
Add before this line:
```
---
IDE
---
```
Greetings, Daria

Reply:
Thank you Daria, I'll add it right away :)